app/controllers/api/v2/courses_controller.rb

```diff
@@ -170,7 +171,11 @@ def course_params
             :science,
             :qualification,
             :age_range_in_years,
+<<<<<<< HEAD
```

Comment:
Looks like you've got some merge conflicts still remaining from rebasing master back in :eyes:

Reply:
Done 👍